src/integTest/groovy/net/wasdev/wlp/gradle/plugins/AbstractIntegrationTest.groovy

```diff
@@ -50,8 +43,8 @@ abstract class AbstractIntegrationTest {
             }
         }
     }
-
-    protected static File createTestProject(File parent, File sourceDir, String buildFilename) {
+    
+    protected static File createTestProject(File parent, File sourceDir) {
```

Comment:
I haven't discussed with him yet as he's away - I believe 'buildFilename' is not being used anywhere and we can safely remove the redundant parameter. I also cleaned up other codes  where it's referenced.